scripts/py_matter_idl/matter_idl/test_idl_generator.py

```diff
@@ -55,8 +55,8 @@ def ReadMatterIdl(repo_path: str) -> str:
         return stream.read()
 
 
-def ParseMatterIdl(repo_path: str, skip_meta: bool) -> Idl:
-    return CreateParser(skip_meta=skip_meta).parse(ReadMatterIdl(repo_path))
+def ParseMatterIdl(repo_path: str, skip_meta: bool, merge_globals: bool) -> Idl:
```

Comment:
Is this change related to the title of this PR?

Reply:
No, I think it was a master merge. Updated with a master merge.